org.eclipse.jdt.core/formatter/org/eclipse/jdt/internal/formatter/CommentsPreparator.java

```diff
@@ -1250,9 +1386,14 @@ private boolean tokenizeMultilineComment(Token commentToken) {
 						i++;
 					position = i + 1;
 				} else if (!ScannerHelper.isWhitespace(c)) {
-					while (this.tm.charAt(i) == markerChar && lineBreaks > 0)
+					int markDownTaglen = 0;
```

Comment:
`markdownTag` can be understood as `@return` or something, I'd go with `markerCharCount`.